### PR TITLE
Add minio service to k8s-env 

### PR DESCRIPTION
**Pull request by sotojn**
This PR makes the following changes:

- Adds a working `minio` to local k8s environment
  - Running the `yarn k8s` command now has the added option of using minio as a service
- Adds `--asset-storage` option flag for k8s command that allows the user to chose which asset storage backend to use
- Implements error handling for conflicting k8s options
- Bumps **@terascope/scripts** from `v0.73.0` to `v0.74.0`

Ref to issue #3573

### CODE REVIEW

File: packages/teraslice/test/services/assets-spec.ts [hunk 1]
```diff
@@ -43,11 +44,12 @@ describe('Assets Service', () => {
...
     context.sysconfig.teraslice.api_response_timeout = 30000;
-    /// Setting port for the asset service
-    process.env.port = '55678';
-    const service = new AssetsService(context);
+    let service: AssetsService;
 
     beforeAll(async () => {
+        /// Setting port for the asset service
+        process.env.port = String(await findPort());
+        service = new AssetsService(context);
```
Comment (godber) [inline packages/teraslice/test/services/assets-spec.ts:52]: Why are these changes here?

Comment (sotojn) [reply packages/teraslice/test/services/assets-spec.ts:52]: This was something I threw in to potentially resolve that test from failing randomly in CI. I should just take this out, write an issue and make a separate PR for this.

Comment (godber) [reply packages/teraslice/test/services/assets-spec.ts:52]: Let's pull this out since it's unrelated and I think we want to talk more about it.  At least I don't want to evaluate it in the context of this pr.

**Review by godber — APPROVED** [review]
